### PR TITLE
document host resource settings for large networks

### LINKED ISSUE #192: document host resource settings
for example, `ulimit -n 4096` and these lines in `/etc/sysctl.conf`:

```
net.ipv4.neigh.default.gc_thresh1 = 80000
net.ipv4.neigh.default.gc_thresh2 = 90000
net.ipv4.neigh.default.gc_thresh3 = 100000
fs.inotify.max_user_watches = 100000

```

### PR DESCRIPTION
**Pull request by willcl-ark**
closes: #192

ping @pinheadmz

### CODE REVIEW

File: docs/running.md [hunk 1]
```diff
@@ -11,4 +11,59 @@ To start the server in the foreground simply run:
...
+
+# gc_thresh2 - Set the soft threshold for garbage collection to initiate ARP entry clean up
+net.ipv4.neigh.default.gc_thresh2 = 90000
+
+# gc_thresh3 - Set the hard threshold beyond which the system will start to drop ARP entries
+net.ipv4.neigh.default.gc_thresh3 = 100000
+
+# Increase inotify watchers limit to allow more files to be monitored for changes
+# This is beneficial for applications like file sync services, IDEs or web development servers
+fs.inotify.max_user_watches = 100000
```
Comment (pinheadmz) [inline docs/running.md:33]: I had to add this one too:
`fs.inotify.max_user_instances = 100000`

I was getting "too many open files" errors in the cadvisor tank

Comment (willcl-ark) [reply docs/running.md:33]: added

Comment (willcl-ark) [reply docs/running.md:33]: It also goes in /etc/sysctl.conf right?

Comment (pinheadmz) [reply docs/running.md:33]: Yeah!

**Review by pinheadmz — APPROVED** [review]

File: docs/running.md (hunk 1 above)
Comment (willcl-ark) [reply docs/running.md:33]: noice. Want anything else in here?

Comment (willcl-ark) [reply docs/running.md:33]: I guess not!

**Pull request merged**